src/features/joinForms/panes/JoinSubmissionPane.tsx

```diff
@@ -56,18 +57,28 @@ const JoinSubmissionPane: FC<Props> = ({ orgId, submissionId }) => {
           value={data.form.title}
         />
       </Box>
-      <Box display="flex" gap={1} justifyContent="stretch" my={4}>
-        <Button sx={{ flexGrow: 1 }} variant="outlined">
-          {messages.submissionPane.rejectButton()}
-        </Button>
-        <Button
-          onClick={() => approveSubmission()}
-          sx={{ flexGrow: 1 }}
-          variant="contained"
-        >
-          {messages.submissionPane.approveButton()}
-        </Button>
-      </Box>
+      {!data.accepted ? (
```

Comment:
I am awaiting the design here – if the buttons and the person link are not to be rendered in the same space (which might be the outcome), this issue will pretty much resolve itself I think

Reply:
Ok! And otherwise, just replace with this pattern:

```tsx
{data.accepted && (
  <h1>ACCEPTED!</h1>
)}
{!data.accepted && (
  <h1>NOT ACCEPTED</h1>
)}
```